zerver/lib/zcommand.py

```diff
@@ -30,4 +35,26 @@ def process_zcommands(content: str, user_profile: UserProfile) -> Dict[str, Any]
         ret = dict(msg=msg)
         return ret
 
+    if command == 'subscribe':
+        linked_stream = possible_linked_stream_names(content)
+        mentioned_user = MentionData(user_profile.realm.id, content).get_user_ids()
+
+        if len(tokens) < 3 or len(linked_stream) != 1 or len(mentioned_user) > 1:
+            ret = dict(msg="Usage: /subscribe #<stream_name> @<user>")
```

Comment:
I've not tried the code, but does this also work with the latter parameters switched, since presumably the first two components are just looking for a stream and a name? So does the help text strictly apply?

Reply:
It doesn't strictly apply. It goes same for `/mute_topic` which I have mentioned in #9754, but I forgot to mention it here. Both `/subscribe #<stream_name> @<user>` and `/subscribe @<user> #<stream_name>` would work.

Reply:
Do you plan to update this help text? Or maybe add a note in the code somewhere?